lib/mixins/state-machine.js

```diff
@@ -24,7 +24,9 @@ module.exports = function StateMachine(Model, settings) {
       // Persist the updated state in the database.
       onenter: function onenter(options) {
         debug(`Finalizing state: ${options.to}`)
-        return options.instance.updateAttribute(stateProperty, options.to)
+        const transitionOptions = options.transitionOptions || { }
```

Comment:
How does this let you pass in `transitionOptions`? Does it work for the following situations:

1) `Model.activate({ skipSomething: true }`

2) With default transition options defined in the JSON, eg:
  ```
"mixins": {
  "StateMachine": {
    "stateProperty": "status",
    "events": [
      { "name": "activate", "from": "none", "to": "active", transactionOptions: { skipSomething: true}  },
      { "name": "cancel", "from": "active", "to": "canceled" },
      { "name": "reactivate", "from": "canceled", "to": "active" },
      { "name": "expire", "from": [ "active", "canceled" ], "to": "expired" }
    ]
  }
}
  ```

Could you update with README with details of how this can be used?